src/diffusers/models/cross_attention.py

```diff
@@ -463,6 +463,49 @@ def __call__(self, attn: CrossAttention, hidden_states, encoder_hidden_states=No
         return hidden_states
 
 
+class TorchAttentionProcessor:
```

Comment:
Can we maybe call it 2_0 since we'll soon rename `CrossAttnProcessor` to `AttnProcessor`?
```suggestion
class AttnProcessor2_0:
```

Reply:
`Torch2_0AttentionProcessor` (I know I will be buried alive in hell for disregarding all the Python standards).

Reply:
The `Torch` prefix doesn't make much sense here IMO since everything is obviously PyTorch code. 

Still advocating for `AttnProccesor2_0` since we'll probs rename `CrossAttnProcessor` to `AttnProcessor`